src/app/components/Navbar/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import clsx from 'clsx';
+import { useHistory } from 'react-router-dom';
+
+import { removeStorage } from '~utils/storage';
+import ROUTES from '~constants/routes';
+import wolox from '~assets/logos/wolox.png';
+
+import styles from './styles.module.scss';
+
+export default function Navbar() {
+  const history = useHistory();
+
+  const handleSignOut = () => {
+    removeStorage('accessToken');
+    history.push(ROUTES.base);
+    history.go();
+  };
+
+  return (
+    <div className={styles.container}>
+      <div className={clsx('row space-between middle', styles.content)}>
```

Comment:
i think this div is not necessary, you can use a single div for this
Also we only use `clsx` if there is a condition
```
<div className={`row space-between middle ${styles.container}`}>
```

Reply:
I am using two divs because max-width is being used, so that the proportions are maintained on large screens